Taarafo.Core/Services/Foundations/Posts/PostService.cs

```diff
@@ -38,5 +39,15 @@ public partial class PostService : IPostService
 
         public IQueryable<Post> RetrieveAllPosts() =>
         TryCatch(() => this.storageBroker.SelectAllPosts());
+
+        public ValueTask<Post> RetrievePostByIdAsync(Guid postId) =>
+            TryCatch(async () =>
+            {
+                ValidatePostById(postId);
+                Post storagePost = await this.storageBroker.SelectPostByIdAsync(postId);
```

Comment:
This line extends beyond the definition of the method - so please break it a bit. with a space before and after since it becomes a multi-liner.